index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+        </div>
+    </div>
+    <div id="tribute-info"></div>
+</div> 
+   
```

Comment:
Please use HTML5 semantics tags (for example, header, nav) on the divs. Some benifits of using these tags are :

- Provide your page with a Search Engine Optimization (SEO) score and
- Ease code navigation when updating features or troubleshooting bugs